clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/agent/KubernetesServerGroupCachingAgent.groovy

```diff
@@ -0,0 +1,237 @@
+/*
+ * Copyright 2016 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.kubernetes.provider.agent
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.frigga.Names
+import com.netflix.spectator.api.Registry
+import com.netflix.spinnaker.cats.agent.AccountAware
+import com.netflix.spinnaker.cats.agent.AgentDataType
+import com.netflix.spinnaker.cats.agent.CacheResult
+import com.netflix.spinnaker.cats.agent.CachingAgent
+import com.netflix.spinnaker.cats.agent.DefaultCacheResult
+import com.netflix.spinnaker.cats.provider.ProviderCache
+import com.netflix.spinnaker.clouddriver.cache.OnDemandAgent
+import com.netflix.spinnaker.clouddriver.cache.OnDemandMetricsSupport
+import com.netflix.spinnaker.clouddriver.kubernetes.cache.Keys
+import com.netflix.spinnaker.clouddriver.kubernetes.deploy.KubernetesUtil
+import com.netflix.spinnaker.clouddriver.kubernetes.KubernetesCloudProvider
```

Comment:
Generally try to prefer ascii sort-order for imports (
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing
).

On Fri, Feb 5, 2016 at 12:24 PM Lars Wander <notifications@github.com>
wrote:

> In
> clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/provider/agent/KubernetesServerGroupCachingAgent.groovy
> <https://github.com/spinnaker/clouddriver/pull/276#discussion_r52045604>:
>
> > +package com.netflix.spinnaker.clouddriver.kubernetes.provider.agent
> > +
> > +import com.fasterxml.jackson.databind.ObjectMapper
> > +import com.netflix.frigga.Names
> > +import com.netflix.spectator.api.Registry
> > +import com.netflix.spinnaker.cats.agent.AccountAware
> > +import com.netflix.spinnaker.cats.agent.AgentDataType
> > +import com.netflix.spinnaker.cats.agent.CacheResult
> > +import com.netflix.spinnaker.cats.agent.CachingAgent
> > +import com.netflix.spinnaker.cats.agent.DefaultCacheResult
> > +import com.netflix.spinnaker.cats.provider.ProviderCache
> > +import com.netflix.spinnaker.clouddriver.cache.OnDemandAgent
> > +import com.netflix.spinnaker.clouddriver.cache.OnDemandMetricsSupport
> > +import com.netflix.spinnaker.clouddriver.kubernetes.cache.Keys
> > +import com.netflix.spinnaker.clouddriver.kubernetes.deploy.KubernetesUtil
> > +import com.netflix.spinnaker.clouddriver.kubernetes.KubernetesCloudProvider
>
> Isn't 'K' > 'c'?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/clouddriver/pull/276/files#r52045604>.
>